operators/velodyne_lidar/cpp/velodyne_convert_xyz.cu

```diff
@@ -0,0 +1,160 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Ported from NVIDIA DeepMap SDK
+
+#include "velodyne_constants.hpp"
+#include "velodyne_convert_xyz.hpp"
+
+#include <algorithm>
+#include <iostream>
+
+#include <holoscan/holoscan.hpp>
+
+// Defines all tables.
+// Use constant memory cause it's 100X faster than shared memory.
+__constant__ double d_vlp16_sin_pitch[data_collection::sensors::kVLP16LineCount];
+__constant__ double d_vlp16_cos_pitch[data_collection::sensors::kVLP16LineCount];
+
+// Quick return cuda error and error message. Will force quit program cause all
+// cuda errors are fatal errors.
+#define CHECK_CUDA_ERROR(err) (HandleError(err, __FILE__, __LINE__))
```

Comment:
Could you use the same CUDA_TRY macro as in the other file below for consistency?

Reply:
Fixed